src/main/java/nl/recognize/dwh/application/loader/AbstractEntityLoader.java

```diff
@@ -78,6 +79,11 @@ public void applyFilters(QueryBuilder queryBuilder, List<RequestFilter> filters)
         }
     }
 
+    @Override
+    public String getIdentifierType() {
+        return FieldMapping.TYPE_UUID;
```

Comment:
Is de default nu UUID geworden? Wil je dat wel (ivm backwards compat), zou dit eigenlijk niet TYPE_ID moeten blijven? Deze lib wordt nl ook gebruikt door AssetCheck..

Reply:
Je hebt helemaal gelijk. Was er niet helemaal bij ;-)